tests/E2E/test/globals.webdriverio.js

```diff
@@ -16,9 +16,9 @@ global.URL = siteUrl;
 global.install_language = argv.LANGUAGE || 'en';
 global.install_country = argv.COUNTRY || 'france';
 global.country_currency = argv.CURRENCY || 'EUR';
-global.db_server = argv.DB_SERVER || 'lolcahost';
+global.db_server = argv.DB_SERVER || 'localhost';
 global.db_user = argv.DB_USER || 'root';
-global.db_passwd = argv.DB_PASSWD || 'doge';
+global.db_passwd = argv.DB_PASSWD || 'root';
```

Comment:
I'm not in favor to use root as defaut password, it's a bad practice, we shouldn't use this kind of value as default :/

Reply:
Fixed :+1: